substra/cli/interface.py

```diff
@@ -148,6 +148,17 @@ def click_option_verbose(f):
     )(f)
 
 
+def click_option_permissions(f):
+    """Add permissions file path to command."""
+    return click.option(
+        '--permissions-path',
+        'permissions',
+        type=click.Path(exists=True, dir_okay=False), callback=load_json_from_path,
+        metavar="PATH",
```

Comment:
I'm not sure but I think metavar only work with arguments, not options, could you double check against the documentation?

Reply:
Indeed, it was a mistake